src/views/virtualmachines/details/tabs/details/utils/gridHelper.tsx

```diff
@@ -83,11 +81,13 @@ export const getRunningVMRightGridPresentation = (
       ? GuestAgentIsRequiredText
       : guestAgentData?.timezone?.split(',')[0],
     node: <ResourceLink kind={NodeModel.kind} name={nodeName} />,
-    userCredentials: sshServiceLoaded ? (
-      <UserCredentials vmi={vmi} sshService={sshService} />
+    sshAccess: sshServiceLoaded ? (
```

Comment:
This logic is duplicated in the `SSHDetails` component. You could create a small component for it that takes `sshServiceLoaded` as a prop and then use it in both places. Not necessary, but could make it a bit simpler.